app/forms/application_form.rb

```diff
@@ -1,7 +1,9 @@
 class ApplicationForm
   include ActiveModel::Model
+  include ActiveModel::Attributes
   include ActionView::Helpers::TranslationHelper
   include Rails.application.routes.url_helpers
+  include DateAttributes
```

Comment:
Is it more beneficial to standardise the tooling available for our `FormObjects`?

I'd also want to add `ActiveModel::Attributes` to the base `ApplicationForm` really, rather than pick and choose which class methods it has access to, but may not use.

Implicit behaviours definitely need to be included case by case though.

So like, having us have a standard `ActiveRecord` base, but choosing whether or not to use the provided tooling, e.g. `belongs_to` and `validates` etc, makes more sense to me.

What do you think?

Reply:
This kind of reaches into the code pollution argument. As no placements forms require this `DateAttributes` concern added, and if added to the base model would be included into every form object.
Given this is also not being introduced across the board, such as with MentorForm which has date attributes, it's my opinion that the `DateAttributes` concern only be added to forms which require it.

I have no issues with adding `ActiveModel::Attributes` as the only form currently used by Placements is Placements::Placements::FilterForm, which includes them anyway.